src/modules/launcher/Wox.Infrastructure/Helper.cs

```diff
@@ -117,5 +117,34 @@ public static Process OpenInConsole(string path)
 
             return Process.Start(processStartInfo);
         }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Suppressing this to enable FxCop. We are logging the exception, and going forward general exceptions should not be caught")]
+        public static bool OpenInShell(string path, string arguments = null, string workingDir = null, bool runAsAdmin = false)
+        {
+            using (var process = new Process())
+            {
+                process.StartInfo.FileName = path;
+                process.StartInfo.WorkingDirectory = string.IsNullOrWhiteSpace(workingDir) ? string.Empty : workingDir;
+                process.StartInfo.Arguments = string.IsNullOrWhiteSpace(arguments) ? string.Empty : arguments;
+
+                if (runAsAdmin)
+                {
+                    process.StartInfo.Verb = "RunAs";
+                }
+
+                process.StartInfo.UseShellExecute = true;
+
+                try
+                {
+                    process.Start();
+                    return true;
+                }
+                catch (System.Exception ex)
```

Comment:
Let's catch specific exceptions. Two possible exceptions here are InvalidOperationException and Win32Exception. If FileName is present then we only need to catch Win32Exception.

Reply:
Since this is using `UseShellExecute = true` catch `Win32Exception` should be fine.